sim/druid/moonfire.go

```diff
@@ -0,0 +1,87 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (druid *Druid) registerMoonfireSpell() {
+	// TODO: Genesis increase to Moonfire duration
+	// TODO: Shooting stars proc on periodic damage
+	// TODO: Glyph of Moonfire increase to periodic damage
+	// TODO: Calculate ticks based on haste
```

Comment:
for this todo just set the AffectedByCastSpeed in the Dot config

Reply:
fixed